lib/router/router.js

```diff
@@ -63,6 +63,13 @@ function getTransitionByIntent(intent, isIntermediate) {
 
   // Abort and usurp any previously active transition.
   if (this.activeTransition) {
```

Comment:
Can we save off a reference to `this.activeTransition`? Would help keep some of the lines below shorter.